app/src/test/java/uk/ac/ebi/atlas/search/ToBeDeprecatedJsonGeneSearchControllerWIT.java

```diff
@@ -343,14 +343,12 @@ void whenSearchTermNotExistsInDBThenCellTypeSearchReturnsEmptySet() throws Excep
 
     @Test
     void whenSearchTermExistsInDBThenReturnsSetOfCellType() throws Exception {
-        var shouldBeGeneThatPartOfExperiments = "AT2G21840";
-
-        var expectedCellType = "trichoblast 17";
+        var shouldBeGeneThatPartOfExperiments =
+                jdbcTestUtils.fetchRandomGeneFromSingleCellExperiment("E-CURD-4");
```

Comment:
use can use this method to get random experiment accession - `fetchRandomExperimentAccession()`,  we have a few ways to get random experiment accession in the` JDBCutils class`, you can use one based on your need here..

Reply:
ok, I randomised the experiment accession.